dss/api/files.py

```diff
@@ -112,6 +116,25 @@ def get_helper(uuid: str, replica: Replica, version: str=None, token: str=None):
 def _verify_checkout(
         replica: Replica, token: typing.Optional[str], file_metadata: dict, blob_path: str,
 ) -> typing.Tuple[str, bool]:
+    try:
```

Comment:
I am confused as this is not consistent with the PR title.  Is it possible you mixed commits and PRs?

Reply:
Yea I can see how this is confusing... Essentially I had my branches set up as: `master` -> `create-blob-ttl-env-vars` -> `optimize-file-get` -> `optimize-bundle-get` while keeping the downstream branches up-to-date. In hindsight, `optimize-bundle-get` should have branched off of `create-blob-ttl-env-vars`. Regardless, the file-get related commits will disappear from this PR when https://github.com/HumanCellAtlas/data-store/pull/1516 is merged. Sorry for the confusion!